src/data/faq_de.json

```diff
@@ -1287,7 +1287,7 @@
                         "textblock": [
                             "Es handelt sich bei dieser Benachrichtigung um einen Sicherheitshinweis: Die Corona-Warn-App hat auf Ihrem Gerät Anzeichen für Administrator-Berechtigungen gefunden. Solche Berechtigungen sind auch als Root-Berechtigungen bekannt.",
                             "Sie erhalten diesen Sicherheitshinweis, um darauf aufmerksam gemacht zu werden, dass durch diese Berechtigungen ein erhöhtes Risiko besteht, dass andere Anwendungen auf Ihrem Smartphone auf Daten der Corona-Warn-App zugreifen könnten.",
-                            "Sie werden diesen Hinweis immer dann erhalten, wenn Sie die App neu starten, neu installieren oder updaten.",
+                            "Sie werden diesen Hinweis immer dann erhalten, wenn Sie die App neu starten, neu installieren oder updaten. Diese Warnungen können ab dem Release 2.16 bis zum nächsten Update der App unterbunden werden.",

```

Comment:
```suggestion
                            "Sie werden diesen Hinweis immer dann erhalten, wenn Sie die App neu starten, neu installieren oder updaten. Diese Warnungen können ab dem Release 2.16 bis zum nächsten Update der App unterdrückt werden.",
```

Reply:
@dsarkar 
I agree that the change to "unterdrückt" would be better.

The verb "unterdrücken" is used in the app notice - see https://github.com/corona-warn-app/cwa-app-android/blob/release/2.16.x/Corona-Warn-App/src/main/res/values-de/release_info_strings.xml#L29 and

"Mit Version 2.12 wurde die Anzeige eines Sicherheitshinweises eingeführt, falls auf Ihrem Smartphone Administratorrechte aktiv sind. Diesen Hinweis können Sie nun bis zum nächsten Update unterdrücken."

So this suggested change would make the FAQ better aligned to the app text.

Reply:
Thanks @Ein-Tim  @MikeMcC399, I agree to use same/similar wording in general.